corehq/apps/export/dbaccessors.py

```diff
@@ -66,25 +66,42 @@ def get_form_inferred_schema(domain, app_id, xmlns):
 
 def get_form_export_instances(domain):
     from .models import FormExportInstance
-
     key = [domain, 'FormExportInstance']
     return _get_export_instance(FormExportInstance, key)
 
 
 def get_case_export_instances(domain):
     from .models import CaseExportInstance
-
     key = [domain, 'CaseExportInstance']
     return _get_export_instance(CaseExportInstance, key)
 
 
+def get_brief_form_export_instances(domain):
```

Comment:
I know we often use multiple dbaccessor functions rather than use a series of parameters to control the return values, but in this case I think the latter would simplify things, since there are so many permutations in use.

Reply:
will update

Reply:
17ab783e74c - I did keep one of the methods split into a `get` and `get_brief` version, because it got awkward dealing with objects and dicts in the same method.